nmstatectl/nmstatectl.py

```diff
@@ -38,14 +38,29 @@
 from libnmstate.schema import Route
 
 
-def main():
-    logging.basicConfig(
-        format='%(asctime)s %(name)-12s %(levelname)-8s %(message)s',
-        level=logging.DEBUG,
-    )
+class NmstateEnvVar(object):
+    LOG_LEVEL = 'NMSTATE_LOG_LEVEL'
```

Comment:
I think it is pretty common to use env variables for scripts which may use multiple times a tool. Then they do not need to re-specify the argument on each call.

We can also use this in the future to create a non-root logger for libnmstate (for it to initialize). We do not have it yet, but it may be useful to have one.

Reply:
You mean to use `import logging; log = logging.getLogger(__name__); log.debug(...)` in libnmstate? Makes sense to me. However, we still do not need the environment variable then.

Reply:
You may have missed the first reasoning, which is practically a de-facto standard in all applications I know of. Environment variables are used for scripting the usage of the app, such that the user does not need to repeat itself each time it uses it, add it to the bash profile and so on. Any reasonable app has this option, it costs very little and I see no reasoning not to have it.
In our case, my next suggestion would have been to propose changing the default to `INFO` and enabling `DEBUG` for ourself and the CI through this environment variable.

For the second part, my focus was on the log-level context, meaning: `logging.setLevel()`.
In the future, an app may set the log level for the lib in the code, or the lib providing an env variable which can be used to do the same without any code from the app part (moving it to the app setup/deployment/configuration).